### PR TITLE
Export the C API version via a function

### PR DESCRIPTION
**Pull request by dmbryson**

### CODE REVIEW

File: products/libllbuild/C-API.cpp [hunk 1]
```diff
@@ -25,3 +25,7 @@ const char* llb_get_full_version_string(void) {
 
   return versionString.c_str();
 }
+
+int llb_get_c_api_version(void) {
```
Comment (ddunbar) [inline products/libllbuild/C-API.cpp]: Should this be llb_get_api_version? I guess its obvious it is the C API version?

**Comment by dmbryson** [comment]
@swift-ci please smoke test

**Comment by dmbryson** [comment]
@swift-ci please smoke test

**Comment by dmbryson** [comment]
@swift-ci please smoke test os x

**Comment by dmbryson** [comment]
@swift-ci please smoke test os x

**Comment by dmbryson** [comment]
@swift-ci please smoke test

**Comment by dmbryson** [comment]
@swift-ci please smoke test

**Pull request merged**